lib/processor.js

```diff
@@ -26,7 +28,10 @@ function parse(text) {
         line = 0,
         column = 0,
         mode = modes.NORMAL,
-        next;
+        next,
+        expErr;
+
+    var EXP_ERR_REGEX = /\/\*(?:\s*)error(?:\s+)(.+?)(?:\s*)\*\//g;
```

Comment:
This can be optimized by removing the non-capturing groups: `/\/\*\s*error\s+(.+?)\s*\*\//g`.